test/_features/example_app/complex_form.feature

```diff
@@ -35,8 +35,53 @@ Feature: Complex Form
     Then I should be on the 'select' page showing 'What is the appeal stage?'
     Then I select 'appealStages' and '01. First Tier IAC Appeal - In Country Appeals'
     Then I click the 'Continue' button
+    Then I should be on the 'save-form' page showing 'Would you like to save your form?'
+    Then I select 'No'
+    Then I click the 'Continue' button
     Then I should be on the 'confirm' page showing 'Check your answers before submitting your application.'
     Then I click the 'Confirm submission' button
     Then I should see 'Application sent' on the page
     Then I click the 'Start again' button
     Then I should be on the 'landing-page' page showing 'Choose one of the options below and press continue.'
+
+Scenario: Complex Form Submission With Saving The Form
```

Comment:
Maybe add a tag to this test so it is easier to run this as a single test?

Reply:
added @save_form tag for this test